examples/flash_decoding/example_gqa_decode_varlen_logits.py

```
@@ -34,100 +34,97 @@ def get_configs():
-            T.fill(logsum, 0)
+    valid_block_H = T.min(block_H, kv_group_num)
+
+    Q: T.Tensor(shape_q, dtype)
+    K: T.Tensor(shape_k, dtype)
+    V: T.Tensor(shape_v, dtype)
+    cu_seqlens_k: T.Tensor([batch + 1], T.int32)
+    s_aux: T.Tensor([heads], T.float32)
+    Output = T.empty(shape_o, dtype)
+    S = T.empty(shape_s, dtype)
+
+    with T.Kernel(batch, heads // valid_block_H, num_split, threads=threads) as (bid, hid, bz):
```

Ruff (0.15.7)

[warning] Unpacked variable `bz` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)